students/Gregg/lesson3/mailroom.py

```diff
@@ -179,27 +187,62 @@ def format_row(row_tupl):
     return "{:<27}${:>12.2f} {:>12}  ${:>12.2f}".format(*row_tupl)
 
 
-def list_donations(donor):
+def list_donations(donor_idx):
```

Comment:
this function doesn't seem to list donations, more like `get_donations_by_index`